README.md

```diff
@@ -33,33 +33,97 @@ Examples:
     ~foreman/config/hooks/smart_proxy/after_create/01_email_operations.sh
     ~foreman/config/hooks/audited/adapters/active_record/audit/after_create/01_syslog.sh
 
-## SELinux notes
-
-When using official installation on Red Hat and Fedora system, note that
-SELinux is turned on by default and Foreman is running in confined mode. Make
-sure that hook scripts has the correct context (`foreman_hook_t` on
-RHEL7+/Fedora 19+ or `bin_t` on RHEL6):
-
-    restorecon -RvF /usr/share/foreman/config/hooks
-
-Also keep in mind that the script is running confined, therefore some actions
-might be denied by SELinux. Check audit.log and use audit2allow and other
-tools when writing scripts.
-
 ## Objects / Models
 
 Every object (or model in Rails terms) in Foreman can have hooks.  Check
-`~foreman/app/models` for the full list, but these are the interesting ones:
-
-* `host/managed` (or `host` in Foreman 1.1)
+`~foreman/app/models` for the full list, but these are the interesting ones
+from Foreman 1.11 Core or some its plugins:
```

Comment:
What you have right now is insufficient, after reading the README you have no idea what you can do with hooks. I've shortened the list to four, but I think it's better to have outdated documentation than no documentation.